frontend/src/views/dev_frame/load-mock-data.tsx

```diff
@@ -263,6 +269,70 @@ export function SeedDataMenu({
         }
     }
 
+    async function seedApplications(limit=1000) {
+        setStage("Applications");
+        setProgress(0);
+
+        if (!targetSession) {
+            throw new Error("Need a valid session to continue");
+        }
+
+        count = 0;
```

Comment:
Always use `let` or `const` when creating variables. eslint should be giving you a bunch of warnings in the console about this.